python/chronos/src/bigdl/chronos/data/repo_dataset.py

```diff
@@ -92,3 +96,44 @@ def get_public_dataset(name, path='~/.chronos/dataset', redownload=False, **kwar
                           "Only network_traffic, AIOps, fsi, nyc_taxi, uci_electricity"
                           " uci_electricity_wide "
                           f"are supported in Chronos built-in dataset, while get {name}.")
+
+
+def gen_synthetic_data(len=10000, amplitude=10.0, angular_freq=0.01, scale=1.0,
+                       time_freq="D", **kwargs):
```

Comment:
add a `seed` here? So that we could reproduce the same output.

Reply:
parameters seed and noise_amplitude are added.